google-cloud-translate-v2/README.md

```diff
@@ -0,0 +1,74 @@
+# Ruby Client for the Cloud Translation V2 API
+
+API Client library for the Cloud Translation V2 API
+
+Cloud Translation can dynamically translate text between thousands of language pairs. Translation lets websites and programs programmatically integrate with the translation service.
+
+https://github.com/googleapis/google-cloud-ruby
+
+## Installation
+
+```
+$ gem install google-cloud-translate-v2
+```
+
+## Before You Begin
+
+In order to use this library, you first need to go through the following steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+1. {file:AUTHENTICATION.md Set up authentication.}
+
+## Quick Start
+
+```ruby
+require "google/cloud/translate/v2"
+
+client = ::Google::Cloud::Translate::V2.new
+
+translation = client.translate "Hello world!", to: "la"
+
+puts translation #=> Salve mundi!
+
+translation.from #=> "en"
+translation.origin #=> "Hello world!"
+translation.to #=> "la"
+translation.text #=> "Salve mundi!"
+```
+
+View the [Client Library Documentation](https://googleapis.dev/ruby/google-cloud-translate-v2/latest)
+for class and method documentation.
+
+## Enabling Logging
+
+To enable logging for this library, set the logger for the underlying [gRPC](https://github.com/grpc/grpc/tree/master/src/ruby) library. The logger that you set may be a Ruby stdlib [`Logger`](https://ruby-doc.org/stdlib-2.5.0/libdoc/logger/rdoc/Logger.html) as shown below, or a [`Google::Cloud::Logging::Logger`](https://googleapis.dev/ruby/google-cloud-logging/latest) that will write logs to [Stackdriver Logging](https://cloud.google.com/logging/). See [grpc/logconfig.rb](https://github.com/grpc/grpc/blob/master/src/ruby/lib/grpc/logconfig.rb) and the gRPC [spec_helper.rb](https://github.com/grpc/grpc/blob/master/src/ruby/spec/spec_helper.rb) for additional information.
```

Comment:
gRPC logging won't work in this lib, will it?

Reply:
No it won't! Removed.